lib/IRGen/GenDistributed.cpp

```diff
@@ -659,6 +662,8 @@ void DistributedAccessor::emit() {
   // Witness table for decoder conformance to DistributedTargetInvocationDecoder
   auto *decoderProtocolWitness = params.claimNext();
 
+  (void) params.claimAll();
```

Comment:
This should not be necessary. If we're passing unneeded parameters, we should fix the type signature (as above).